doc/rst/source/events.rst

```diff
@@ -35,6 +35,7 @@ Synopsis
 [ |-W|\ *pen* ]
 [ |SYN_OPT-X| ]
 [ |SYN_OPT-Y| ]
+[ |-Z|\ "custom command" ]
```

Comment:
The "custom command" seems stylistically different to me from other flags. Do you think it's more clear than describing the arguments to **-Z** as *command*. 
```suggestion
[ |-Z|\ *command* ]
```